Sources/MapboxNavigation/StyleManager.swift

```diff
@@ -11,6 +11,14 @@ public protocol StyleManagerDelegate: class, UnimplementedLogging {
      */
     func location(for styleManager: StyleManager) -> CLLocation?
     
+    /**
+     Asks the delegate for the view to be used when refreshing appearance. 
+     
+     The default implementation of this method will attempt to cast the delegate to type
+     `UIViewController` and use its `view` property.
+     */
+    func viewToRefreshOnAppearanceChange(_ styleManager: StyleManager) -> UIView?
```

Comment:
As a general rule, the delegate method’s name should mention the delegating class to avoid potential conflicts with other protocols. A more typical name would be `styleManager(_:viewForApplying:)`, which would take the current Style as an argument.